app/presenters/nsm/check_answers/read_only_report.rb

```diff
@@ -124,6 +124,12 @@ def further_information_section
 
       private
 
+      def case_disposal_form
+        return CaseDisposalCard.new(claim) if claim.include_youth_court_fee.nil?
```

Comment:
Careful, that's not equivalent. The code will only return the card if the value is nil. Your suggested change will return the value if the value is nil or false.

Reply:
Yeah I misread the requirements, this is fine

Reply:
I wonder if better would be `return CaseDisposalCard.new(claim) unless claim.can_claim_youth_court?` i.e. check whether we have asked the user the question rather than whether the user has answered the question.

Reply:
Actually yes, I meant to type that.

We should be checking if it's even viable rather than if they answered it

Reply:
sorry just reading these comments just now as i've been down a rabbit hole trying to make this card backwards compatible with pre-6th dec rep_order_date claims

`claim.can_claim_youth_court?` not sure i can use that as it looks like its excluding BOI 

i've switched this for a combination of feature flag and the rep_order_date - we'll remove the flag post 6th dec but keep the date check on rep_order_date to use old summary card for in reports for pre-6th dec claims - sound ok?

Reply:
Yes it does exclude BOIs which won't go down the flow so will always display the Case Disposal card